networkunit/scores/singularangle.py

```diff
@@ -0,0 +1,106 @@
+"""
+The Singular Angle Similarity (SAS) Score is introduced in Albers et al. (2024)
```

Comment:
```suggestion
The Singular Angle Similarity (SAS) Score is introduced in Albers, Kurth et al. (2024)
```
Looks good to me. I only have one suggestion, citing the paper such that both shared first authors are mentioned.